newm-server/src/main/kotlin/io/newm/server/features/song/SongRoutes.kt

```diff
@@ -36,8 +35,12 @@ fun Routing.createSongRoutes() {
             }
             get {
                 with(call) {
-                    val ownerId = request.queryParameters["ownerId"]?.toUUID() ?: myUserId
-                    respond(repository.getAllByOwnerId(ownerId))
+                    val filter = ownerId?.let { SongFilter.OwnerId(it) }
+                        ?: genre?.let { SongFilter.Genre(it) }
+                        ?: SongFilter.All
+                    respond(
+                        repository.getAll(filter, offset ?: 0, limit ?: 10)
+                    )
```

Comment:
Should we make the page size (limit) biger I seem them usually around 25? Not sure if it matter much

Reply:
That's just the default. The client should not rely on this value and use something like 25 - 30.

Reply:
That's just the default value. Client should specify some value (not relying on default). But I'll make default 25